pkg/oc/clusterup/coreinstall/kubeapiserver/config.go

```diff
@@ -80,9 +82,25 @@ func (opt KubeAPIServerStartConfig) MakeMasterConfig(dockerClient dockerhelper.I
 		return "", err
 	}
 
+	addImagePolicyAdmission(&masterconfig.AdmissionConfig)
+
 	if err := componentinstall.WriteMasterConfig(masterconfigFilename, masterconfig); err != nil {
 		return "", err
 	}
 
 	return masterDir, nil
 }
+
+func addImagePolicyAdmission(admissionConfig *configapi.AdmissionConfig) {
+	// default openshift image policy admission
+	if admissionConfig.PluginConfig == nil {
```

Comment:
so for an out of box "cluster up" invocation this will be nil, right?

So anyone who wants this to work will have to, at a minimum, write the config and put an empty struct in the config for this and then start/restart, right?

Reply:
ignore me.